client-js/app/elements/labrad-nodes.ts

```diff
@@ -253,29 +265,72 @@ export class LabradNodes extends polymer.Base {
   @property({type: Number, value: 0})
   kick: number;
 
+  @property({type: Boolean, value: false, notify: true})
+  isAutostartFiltered: boolean;
+
+  @property({type: Object, value: []})
```

Comment:
"Evaluated at call time

The default argument gets evaluated at call time, so unlike e.g. in Python, a new object is created each time the function is called."

In ECMAScript6, the default parameter doesn't suffer this problem. Do you have a citation for how this is an issue?